packages/builder/src/access-recorder.ts

```diff
@@ -70,6 +70,15 @@ export function computeTemplateAccesses(str?: string, possibleNames: string[] =
     if (typeof (CannonHelperContext as any)[n] === 'function') {
       // the types have been a massive unsolvableseeming pain here
       recorders[n] = _.noop as unknown as AccessRecorder;
+    } else if (typeof (CannonHelperContext as any)[n] === 'object') {
+      for (const o in (CannonHelperContext as any)[n]) {
+        (recorders[n] as any) = {};
+        if (typeof (CannonHelperContext as any)[n][o] === 'function') {
```

Comment:
For these comparisons recommend to use the `_.isFunction`, `_.isPlainObject` and `_.isObject` fns